sigh/bin/sigh

```diff
@@ -67,6 +68,7 @@ class SighApplication
                'The path may be prefixed with a identifier in order to determine which provisioning profile should be used on which app.',
                &multiple_values_option_proc(c, "provisioning_profile", &proc { |value| value.split('=', 2) })
       c.option '-d', '--display_name STRING', String, 'Display name to use'
+      c.option '-g', '--new_bundle_id STRING', String, 'New application bundle ID'
```

Comment:
My doubt about using -b is because it's assigned to the "sigh" command itself
-b, --team_id STRING The ID of your team if you're in multiple teams (SIGH_TEAM_ID) 

So I thought it would be confusing if sigh -b means something and sigh resign -b means another thing.

Also:
-u, --username STRING Your Apple ID Username (SIGH_USERNAME)
-n, --provisioning_name STRING The name of the profile that is used on the Apple Developer Portal (SIGH_PROVISIONING_PROFILE_NAME)
-d, --display_name STRING Display name to use 
-l, --team_name STRING The name of your team if you're in multiple teams (SIGH_TEAM_NAME) 

Well, it seems -e is free...